<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>861</width>
      <height>641</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>10</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>10</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>170</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>170</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>170</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>170</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_16">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>330</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_17">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>330</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_18">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>330</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_19">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>330</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_20">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>330</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_21">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>330</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_22">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_23">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>490</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_24">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>490</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_25">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>490</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_26">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>490</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_27">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>490</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_28">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>490</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>101</width>
       <height>141</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="TeamsLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Equipos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="PointsLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Puntajes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="TeamLabel_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Equipo 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="TeamLabel_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Equipo 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="TeamLabel_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Equipo 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="TeamLabel_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Equipo 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="PointsLabel_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="PointsLabel_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="PointsLabel_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="PointsLabel_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Turno:</string>
    </property>
   </widget>
   <widget class="QLabel" name="playingTeamLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Equipo jugando</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_29">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Evaluar cartas </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1149</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
